postgresql-ha is down size error

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and we'll close it. We
will, however, reopen it if you later provide the information.
-------------------------------
    BUG REPORT INFORMATION
-------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Which chart**: bitnami/postgresql-ha  version 1.4.3, app version 11.6.0

<!-- The name (and version) of the affected chart. -->

**Description**
Pods crash when it's restarted with error:
020-02-17 02:30:23.745 GMT [221] FATAL:  could not connect to the primary server: could not translate host name "postgresql-postgresql-ha-postgresql-1.postgresql-postgresql-ha-postgresql-headless.postgresql.svc.cluster.local" to address: Name or service not known
postgresql-repmgr 02:30:23.75 INFO  ==> ** Starting repmgrd **
2020-02-17 02:30:23.774 GMT [225] FATAL:  could not connect to the primary server: could not translate host name "postgresql-postgresql-ha-postgresql-1.postgresql-postgresql-ha-postgresql-headless.postgresql.svc.cluster.local" to address: Name or service not known
[2020-02-17 02:30:23] [NOTICE] repmgrd (repmgrd 5.0.0) starting up
INFO:  set_repmgrd_pid(): provided pidfile is /opt/bitnami/repmgr/tmp/repmgr.pid
[2020-02-17 02:30:23] [NOTICE] starting monitoring of node "postgresql-postgresql-ha-postgresql-0" (ID: 1000)
[2020-02-17 02:30:23] [ERROR] connection to database failed
[2020-02-17 02:30:23] [DETAIL]
could not translate host name "postgresql-postgresql-ha-postgresql-1.postgresql-postgresql-ha-postgresql-headless.postgresql.svc.cluster.local" to address: Name or service not known

[2020-02-17 02:30:23] [DETAIL] attempted to connect using:
  user=repmgr password=xxx connect_timeout=5 dbname=repmgr host=postgresql-postgresql-ha-postgresql-1.postgresql-postgresql-ha-postgresql-headless.postgresql.svc.cluster.local port=5432 fallback_application_name=repmgr
[2020-02-17 02:30:23] [ERROR] unable connect to upstream node (ID: 1001), terminating
[2020-02-17 02:30:23] [HINT] upstream node must be running before repmgrd can start
<!-- Briefly describe the problem you are having in a few paragraphs. -->

**Steps to reproduce the issue:**

1. Setup postgresql ha with replica = 2
2. Resize replica to 1
3. When postgresql is running only 1 pod, try to delete this pods, it will recreate new pod and crash forever

**Describe the results you received:**

NAME                                               READY   STATUS             RESTARTS   AGE
postgresql-postgresql-ha-pgpool-7c8bd64fb9-wq76h   0/1     Running            0          19h
postgresql-postgresql-ha-postgresql-0              1/2     CrashLoopBackOff   6          7m29s

**Describe the results you expected:**

postgresql-postgresql-ha-postgresql-0 should run. I try to change back replica = 2 but it can't start because pod index 0 don't start, pod index 1 will not start.

<!-- What you expect to happen -->

**Additional information you deem important (e.g. issue happens only occasionally):**
 2020-02-17 02:30:23.774 GMT [225] FATAL:  could not connect to the primary server: could not translate host name "postgresql-postgresql-ha-postgresql-1.postgresql-postgresql-ha-postgresql-headless.postgresql.svc.cluster.local" to address: Name or service not known

When this pod is restarted, repmgrd can't connect to pod postgresql-postgresql-ha-postgresql-0, it auto try connect to pods postgresql-postgresql-ha-postgresql-1, but pod postgresql-postgresql-ha-postgresql-1 can't start if postgresql-postgresql-ha-postgresql-0 don't start in statefulset.
<!-- Any additional information, configuration or data that might be necessary to reproduce the issue. -->


**Version of Helm and Kubernetes**: 3.0

- Output of `helm version`:

```
(paste your output here)
```

- Output of `kubectl version`: 1.15

```
(paste your output here)
```

--- a/bitnami/postgresql-ha/Chart.yaml
+++ b/bitnami/postgresql-ha/Chart.yaml
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql-ha
-version: 1.4.7
+version: 1.4.8
 appVersion: 11.7.0
 description: Chart for PostgreSQL with HA architecture (using Replication Manager (repmgr) and Pgpool).
 keywords:

--- a/bitnami/postgresql-ha/values-production.yaml
+++ b/bitnami/postgresql-ha/values-production.yaml
@@ -51,7 +51,7 @@ postgresqlImage:
 pgpoolImage:
   registry: docker.io
   repository: bitnami/pgpool
-  tag: 4.1.1-debian-10-r6
+  tag: 4.1.1-debian-10-r19
   ## Specify a imagePullPolicy. Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
   ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
   ##

--- a/bitnami/postgresql-ha/values.yaml
+++ b/bitnami/postgresql-ha/values.yaml
@@ -51,7 +51,7 @@ postgresqlImage:
 pgpoolImage:
   registry: docker.io
   repository: bitnami/pgpool
-  tag: 4.1.1-debian-10-r6
+  tag: 4.1.1-debian-10-r19
   ## Specify a imagePullPolicy. Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
   ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
   ##

